Heejung/Lv2/메뉴 리뉴얼.js

```diff
@@ -0,0 +1,38 @@
+function solution(orders, course) {
+  let answer = [];
+  for (let amount of course) {
+    let tmp = {};
+    for (let i = 0; i < orders.length; i++) {
+      // 모든 요소들을 문자열의 오름차순으로 정렬
+      for (let j of combination(orders[i], amount).map((v) =>
+        [...v].sort().join('')
+      )) {
+        tmp[j] = tmp[j] ? tmp[j] + 1 : 1; // 각 조합이 몇번 등장했는지
+      }
+    }
+    // amount에서 기록된 모든 조합 -> 등장횟수의 내림차순 정렬
+    tmp = Object.entries(tmp).sort((a, b) => -a[1] + b[1]);

```

Comment:
```suggestion
    tmp = Object.entries(tmp).sort((a, b) => b[1] - a[1]);
```
제 생각에는 이렇게 표현하는게 더 적절할거 같아요.